optimum/onnxruntime/quantization.py

```diff
@@ -81,71 +84,95 @@ class ORTQuantizer(ABC):
     Handles the ONNX Runtime quantization process for models shared on huggingface.co/models.
     """
 
-    @staticmethod
+    @classmethod
     def from_pretrained(
-        model_name_or_path: Union[str, os.PathLike], feature: str, opset: Optional[int] = None
+        cls,
+        model_name_or_path: Union[str, os.PathLike],
+        use_auth_token: Optional[Union[bool, str, None]] = None,
+        from_transformers: Optional[bool] = False,
+        task: Optional[str] = None,
```

Comment:
should the user provide the `task` or the `feature`. I moved it to the `task` to derive the correct ORTModelForXXX class from the `SUPPORTED_TASK` to make it easier for the user. 
But we could also create a `SUPPORTED_FEATURE` mapping and go with `feature`. 
What do you think @echarlaix @regisss @lewtun ?

Reply:
What would be the difference between `task` and `feature`? Or is this just about proper naming?

Reply:
`task`s are: `text-classificaiton`, `question-answering`,  `text-generation`.... 
`feature`s are: `sequence-classification`, `causal-lm` see: https://huggingface.co/docs/transformers/serialization#selecting-features-for-different-model-topologies

Reply:
my bad it is already `feature` but the attribute name in the ORTModelForXXX was confusing with `pipeline_task`. I changed it to `export_feature`.

Reply:
Ah okay, I must admit this is a bit confusing. Not sure what was the idea behind having these two different sets.
IMO this may be disturbing for users, even if it is properly documented. If adding `feature` does not enable to do anything more, I would keep it as it is.

Reply:
Ah okay then it looks fine to me, thanks!